src/ate/test_programs/dut_lib/dut_lib.h

```diff
@@ -17,17 +17,16 @@ class DutLib {
   void* transport_;
 
  public:
-  DutLib(){};
+  DutLib(void* transport_);
```

Comment:
Consider making this constructor `private`, since the caller probably has no business getting an appropriate `void *` to construct this object.

Reply:
ah good idea. fixed.